src/NexusMods.App.UI/Overlays/UpgradeToPremium/UpgradeToPremiumView.axaml

```diff
@@ -0,0 +1,33 @@
+<reactive:ReactiveUserControl
+    x:TypeArguments="local:IUpgradeToPremiumViewModel"
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:reactive="http://reactiveui.net"
+    xmlns:local="clr-namespace:NexusMods.App.UI.Overlays"
+    mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+    xmlns:base="clr-namespace:NexusMods.App.UI.Overlays.Generic.MessageBox.Base"
+    x:Class="NexusMods.App.UI.Overlays.UpgradeToPremiumView">
+
+    <base:MessageBoxBackground>
```

Comment:
I hate that we are still stuck with these overlays and have no plans to rectify this